src/main/java/org/jfrog/hudson/pipeline/common/docker/DockerImage.java

```diff
@@ -134,12 +134,12 @@ public Module generateBuildInfoModule(Run build, TaskListener listener, Artifact
         try {
             dependenciesClient = server.createArtifactoryDependenciesClient(
                     preferredResolver.provideUsername(build.getParent()), preferredResolver.providePassword(build.getParent()),
-                    server.createProxyConfiguration(Jenkins.getInstance().proxy), listener);
+                    preferredResolver.provideAccessToken(build.getParent()), server.createProxyConfiguration(Jenkins.getInstance().proxy), listener);
```

Comment:
What is the reason for this change?

Reply:
I had to add this parameter after I changed the constructor to get all 3 (username, password and access-token).
It's irrelevant after the current change.